projects/valtimo/components/src/lib/components/form-io/form-io-address/form-io-address.formio.ts

```diff
@@ -0,0 +1,50 @@
+import { Injector } from '@angular/core';
+import {Components, FormioCustomComponentInfo, registerCustomFormioComponent} from '@formio/angular';
+import { FormIoAddressComponent } from './form-io-address.component';
+import {formIoAddressEditForm} from './form-io-address-edit-form';
+
+function extendDefaultEditForm(defaultEditForm) {
+  const defaultForm = defaultEditForm();
+  const tabsComponent = defaultForm.components.find(component => component.type === 'tabs');
+
+  if (tabsComponent) {
+    tabsComponent.components = [...formIoAddressEditForm.components[0].components]; //To add default tabs, use: '...tabsComponent.components'
+    console.log('tabs: ', tabsComponent.components);
+  } else {
+    defaultForm.components = [
+      {
+        type: 'tabs',
+        key: 'customTabs',
+        components: [
+          ...defaultForm.components.map((component, index) => ({ label: `Tab ${index + 1}`, key: `tab${index + 1}`, components: [component] })),
```

Comment:
please format al your code changes with the prettier config in this project. Also, make sure to run lint to make sure there are not any linting erroes